code/modules/lootpanel/search_object.dm

```diff
@@ -0,0 +1,70 @@
+/**
+ * ## Search Object
+ * An object for content lists. Compacted item data.
+ */
+/datum/search_object
+	/// Client attached to the search_object
+	var/datum/weakref/client_ref
```

Comment:
why is this a weakref aren't we in an owning relationship with the thing that deletes us if the client is deleted